docs/integrations/databases/opentelemetry/couchbase-opentelemetry.md

```diff
@@ -0,0 +1,177 @@
+---
+id: couchbase-opentelemetry
+title: Couchbase - OpenTelemetry Collector
+sidebar_label: Couchbase - OTel Collector
+description: Learn about the Sumo Logic OpenTelemetry App for Couchbase.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<img src={useBaseUrl('img/integrations/databases/couchbase-logo.png')} alt="Thumbnail icon" width="50"/> <img src={useBaseUrl('img/send-data/otel-color.svg')} alt="Thumbnail icon" width="45"/>
+
+[Couchbase](https://developer.couchbase.com/what-is-couchbase/), a modern database for enterprise applications, is a distributed document database with a powerful search engine and in-built operational and analytical capabilities. It brings the power of NoSQL to the edge and provides fast, efficient bidirectional synchronization of data between the edge and the cloud.
+
+The Sumo Logic app for Couchbase helps you monitor activity in Couchbase. The pre-configured dashboards provide insight into the Error, events and HTTP Access pattern that help you understand your clusters.
+
+Couchbase logs are sent to Sumo Logic through OpenTelemetry [filelog receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/filelogreceiver).
+
+This App has been tested with the following version of Couchbase:
+
+Non-Kubernetes: Couchbase version: 7.0.2
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Couchbase-OpenTelemetry/schematics-couchbase.png')} alt="Schematics" />
+
+## Fields creation in Sumo Logic for Couchbase
+
+Following are the [Fields](/docs/manage/fields/) which will be created as part of Couchbase App install if not already present:
```

Comment:
Fields - embedded link does not seem to be working. It should redirect to https://help.sumologic.com/docs/manage/fields/

Reply:
We use relative links within docs. Link will work in your local build.